unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatHUDController.cs

```diff
@@ -0,0 +1,67 @@
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine.Events;
+using ChatMessage = ChatController.ChatMessage;
+using ChatMessageType = ChatController.ChatMessageType;
+public class ChatHUDController : IDisposable
```

Comment:
I'm a bit confused as to why we have this class and `WorldChatWindowHUDController`, is the 'ChatHUD' a UI element inside the 'WorldChatWindowHUD'?

Reply:
Yes, the idea is that the ChatHUD can be put inside the friends window when we get to it. I arranged the ChatHUD components so they are agnostic of the context of the HUD element.

The WorldChatWindow has the ChatHUD and the tabs for filtering world/PM, close button, etc.

Reply:
I just saw this info in the description, i'm resolving this.